project.clj

```diff
@@ -11,7 +11,7 @@
                  [honeysql "0.9.4"]]
   :javac-options ["-target" "1.7", "-source" "1.7"]
   :aliases {"bikeshed" ["bikeshed" "--max-line-length" "118" "--var-redefs" "false"]
-            "lint" ["do" ["eastwood"] "bikeshed" "docstring-checker" "check-namespace-decls"]
+            "lint" ["do" ["eastwood"] ["bikeshed"] ["docstring-checker"] ["check-namespace-decls"]]
```

Comment:
Fixed.
The args "docstring-checker" and "check-namespace-decls" were being passed into "bikeshed".

Separate commands need to end in a comma or be in separate vectors. e.g. https://github.com/technomancy/leiningen/blob/master/sample.project.clj#L242
See https://github.com/technomancy/leiningen/blob/master/src/leiningen/do.clj#L21

Reply:
yup, makes sense. Thanks